Wordl/Base.lproj/Main.storyboard

```diff
@@ -8,20 +8,189 @@
+                        <subviews>
+                            <stackView opaque="NO" contentMode="scaleToFill" axis="vertical" translatesAutoresizingMaskIntoConstraints="NO" id="dQz-jh-gFz">
+                                <rect key="frame" x="10" y="374.5" width="394" height="201"/>
+                                <subviews>
+                                    <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="Ahc-mE-fzZ">
+                                        <rect key="frame" x="0.0" y="0.0" width="394" height="67"/>
+                                        <inset key="imageEdgeInsets" minX="0.0" minY="0.0" maxX="2.2250738585072014e-308" maxY="0.0"/>
+                                        <state key="normal" title="Button"/>
+                                        <buttonConfiguration key="configuration" style="plain" title="Start" buttonSize="large">
+                                            <backgroundConfiguration key="background"/>
+                                            <fontDescription key="titleFontDescription" style="UICTFontTextStyleTitle0"/>
+                                        </buttonConfiguration>
+                                        <connections>
+                                            <action selector="startButtonHandleTap:" destination="dmQ-MB-BIN" eventType="touchUpInside" id="Lum-MI-Rfi"/>
+                                        </connections>
+                                    </button>
+                                    <button opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="5ce-e5-BSw">
+                                        <rect key="frame" x="0.0" y="67" width="394" height="67"/>
+                                        <inset key="imageEdgeInsets" minX="0.0" minY="0.0" maxX="2.2250738585072014e-308" maxY="0.0"/>
+                                        <state key="normal" title="Button"/>
+                                        <buttonConfiguration key="configuration" style="plain" title="High scores" buttonSize="large">
+                                            <backgroundConfiguration key="background"/>
+                                            <fontDescription key="titleFontDescription" style="UICTFontTextStyleTitle0"/>
+                                        </buttonConfiguration>
+                                        <connections>
+                                            <action selector="scoresButtonHandleTap:" destination="dmQ-MB-BIN" eventType="touchUpInside" id="e9B-Mo-4ag"/>
+                                            <action selector="startButtonHandleTap:" destination="dmQ-MB-BIN" eventType="touchUpInside" id="rml-rt-svv"/>
```

Comment:
You have two actions on one button. That's why it opens two screens
<img width="1408" alt="Screenshot 2022-05-26 at 16 25 32" src="https://user-images.githubusercontent.com/25828050/170496995-2bba82b6-57d6-4969-a053-c83a62d2e03b.png">